environment.yml

```diff
@@ -14,4 +14,5 @@ dependencies:
   - coverage
   - requests
   - pandas
```

Comment:
The examples I've seen all have:
```
 - pip
 - pip:
    - omegaconf
    - etc
```
so maybe we should follow convention and put `pip` in there even though we can be pretty sure it will always be included.